waterbox/vb/vip.cpp

```diff
@@ -1262,6 +1268,26 @@ static void CopyFBColumnToTarget_HLI(void)
 	}
 }
 
+static void CopyFBColumnToTarget_OnlyLeftRight(void)
+{
+	const int lr = (VB3DMode == VB3DMODE_ONLYLEFT ? 0 : 1);
+
+	if (!DisplayActive)
+	{
+		if (!lr)
+			CopyFBColumnToTarget_SideBySide_BASE(0, 0, 0);
+		else
+			CopyFBColumnToTarget_SideBySide_BASE(0, 1, 0);
+	}
+	else
+	{
+		if (!lr)
+			CopyFBColumnToTarget_SideBySide_BASE(1, 0, 0);
+		else
+			CopyFBColumnToTarget_SideBySide_BASE(1, 1, 0);
+	}

```

Comment:
Is there a reason why this isn't just
```suggestion
  CopyFBColumnToTarget_SideBySide_BASE(DisplayActive, VB3DMode != VB3DMODE_ONLYLEFT, 0);
```

Reply:
I would generally agree that it would be simpler and cleaner, but I am also strongly for sticking to existing conventions and coding style, and this is what the other functions in this source look like. I don't know why they're written like this, but I don't want my added function to look different.